modules/vpc/user_management.yml.tmpl

```diff
@@ -0,0 +1,49 @@
+---
+Consul:
+  Server: 127.0.0.1:8500
+  Namespace: "environments/${environment}/global/nubis-users"
+
+AWS:
+  Region: "${region}"
+  SMTPFromAddress: "${smtp_from_address}"
+  SMTPUsername: "${smtp_username}"
+  SMTPPassword: "${smtp_password}"
+  SMTPHostname: "${smtp_host}"
+  SMTPPort:     "${smtp_port}"
+  AWSIgnoreUserList:
+    - nubis-bootstrap
+    - datadog
+    - gozer
```

Comment:
Yup so in that instance we will need to add that particular instance to the list of ignored users